compiler/runtime/OMRCodeCacheManager.cpp

```diff
@@ -1041,6 +1046,7 @@ OMR::CodeCacheManager::carveCodeCacheSpaceFromRepository(size_t segmentSize,
    uint8_t* start = NULL;
    uint8_t* end = NULL;
    size_t freeSpace = 0;
+   bool shouldAllowCarving;
```

Comment:
Nitpick but since `shouldAllowCarving` is only used in one place, it should be declared below.

Reply:
I am also using `shouldAllowCarving` in the verbose log statement below for future debugging.

Reply:
`         TR_VerboseLog::writeLineLocked(TR_Vlog_FAILURE, "failed to carve code cache of size=%zu from the repository. Free space in code cache repository= %zu. isSufficientPhysicalMemoryAvailableForAllocation = %s", codeCacheSizeToAllocate, freeSpace, shouldAllowCarving? "true":"false");`

Reply:
Ah I see; makes sense.